src/main/java/core/Common.java

```diff
@@ -0,0 +1,136 @@
+package core;
+
+import dukexception.DukeException;
+import task.Task;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+/**
+ * Interact with model and storage to execute the command,
+ * construct response text to be display.
+ */
+public class Common {
```

Comment:
Why is it called Common? Maybe a better name would be Controller since it controls the model and updates view